dashboard/locales/zh-TW/account.json

```diff
@@ -84,14 +84,14 @@
     },
     "referrals": {
         "title": "推廣",
-        "sub-title": "分享你的連結來賺取積分",
-        "description-1": "通過分享你的連結，你將有機會賺取在 Zeabur 上抵用的積分。",
-        "description-2": "對每一個接受你邀請的朋友，當他們升級到了某個付費方案，你將會得到五美金的現金抵用。",
+        "sub-title": "分享你的連結來賺取免費額度",
```

Comment:
Credit should not translated as "額度" (quota).

Reply:
I translated this way because it's basically give people "free money" used to pay for cost on Zeabur.  Though there are some real use cases of "積分", but they usually let people use their credit to redeem specific kinds of products and/or to replace an amount of cash on paying, rather than adding into balance in 1:1 directly.  Furthermore, "credit" can mean "money in bank account" (Cambridge Dictionary), i.e. "balance" ("餘額"), so I think "額度" is okay here.

Reply:
@yuaanlin How do you think? I wonder if people will deposit their "credit" in their payment methods.